src/client/add-metainfo-releases

```diff
@@ -0,0 +1,60 @@
+#!/usr/bin/python3
+
+import argparse
+import re
+import shlex
+import subprocess
+import time
+import xml.etree.ElementTree as ET
+
+
+# Pretend like this is a little bit functional
+def element(tag, text=None, children=(), **kwargs):
+    tag = ET.Element(tag, kwargs)
+    tag.text = text
+    tag.extend(children)
+    return tag
+
+
+def points_from_body(body):
+    for text in re.split('^ *-', body, flags=re.MULTILINE):
+        if point := ' '.join(text.split()).strip():
+            yield element('li', point)
+
+
+def format_release(blob):
+    version, date, body = blob.split('|', 2)
+    url = element('url', f'https://cockpit-project.org/blog/cockpit-{version}.html', type='details')
+    ul = element('ul', children=points_from_body(body))
+    description = element('description', children=(ul,))
+    return element('release', version=version, date=date, children=(url, description))
+
+
+def get_releases():
+    format = '%(if)%(taggerdate)%(then)%(refname:short)|%(taggerdate:short)|%(contents:body)%(end)'
+    output = subprocess.check_output(['git', 'for-each-ref', '--shell', '--sort=-taggerdate',
+                                      '--contains', '258', '--format', format, 'refs/tags'], text=True)
```

Comment:
What's the magic "258" here? Shouldn't that be the `--version` value?

Reply:
This is when we started shipping the flatpak.

Reply:
Added a comment about it.

Reply:
Oh, I see -- so for each release, we have to regenerate the entire release history. I forgot that, but makes sense now.